# Medidas Básicas de la estadística predictiva.


Habitualmente necesitamos disponer de un valor numérico que represente la disparidad de datos de una distribución de frecuencias. Estos valores son los llamados parámetros centrales o medidas de centralización, ya que son valores "intermedios" que se situan alrededor del centro de la distribución.
Se trata de la **media, la moda y la mediana** .

# La media aritmética

Para calcular la media se **suman** todos los valores de los datos y se **divide** por el número total.

Cuando los datos se repiten, es más fácil formar la tabla de frecuencias y sumar los productos de cada valor por las veces que se repite, después dividimos por el nº total de datos.

In [1]:
import numpy as np
from scipy import stats

In [42]:
edad= np.random.randint(0,90,size = 40000)
edad

array([ 5, 27,  0, ..., 57, 58, 32])

In [43]:
media = edad.mean()
print("Podemos observar que la media de nuestro conjunto de datos edad es {} , con la funcion .mean lo podemos calcular sin necesidad de hacer todas las operaciones".format(media))

Podemos observar que la media de nuestro conjunto de datos edad es 44.51175 , con la funcion .mean lo podemos calcular sin necesidad de hacer todas las operaciones


# La mediana

Suponiendo que todos los datos están ordenados la mediana es el valor que ocupa la posición central, de modo que la mitad de los datos son menores y la otra mitad son mayores.

Cuando la variable es discreta la mediana es el primer valor cuya frecuencia acumulada es mayor que **n/2.**

Cuando los datos están agrupados, buscaremos la clase mediana que es la que su frecuencia absoluta acumulada sobrepasa la mitad de los datos (n/2).

In [46]:
mediana = np.median(edad)
mediana

45.0

# La moda


La moda de una variable estadística es el **valor más repetido**, el que tiene mayor frecuencia absoluta.

Si la variable es discreta se busca el valor de mayor frecuencia.
Si los datos están agrupados, la clase de mayor frecuencia se llama clase modal.

In [44]:
moda = stats.mode(edad)
moda

ModeResult(mode=array([53]), count=array([502]))

Podemos observar que en el conjunto de datos obtenidos la moda es 53 & aparece 502 veces en nuestro conjunto de datos.

# CUANTILES

Consideremos  que x1, . . . , xn es un conjunto de n observaciones de una cierta variable cuantitativa de interés, y que estos valores se ordenan de menor a mayor, conservando las repeticiones. Un cuantil es un **número que separa a los datos en dos partes:** un cierto porcentaje de los datos son menores o iguales al cuantil y el porcentaje complementario corresponde a datos que son mayores o iguales al cuantil.

Estos sirven par entender como es la distribucion de un conjunto de datos numericos.

## TIPOS :

Cuando **P “ 0.25,0.50 ó 0.75**, a los cuantiles correspondientes se le llama **CUARTILES**, y se usan las expresiones: primer cuartil, segundo cuartil y tercer cuartil, respectivamente.

Cuando **P “ 0.1,0.2,...,0.9**, a los cuantiles correspondientes se les llama **DECILES**. Podemos referirnos al primer decil de un conjunto de datos, al segundo decil, etcétera.

# Percentil- (Cuantil al 100P%)

El percentil es una medida de posición usada en estadística que indica, una vez ordenados los datos de menor a mayor, el **valor de la variable** por debajo del cual se encuentra un porcentaje dado de observaciones en un grupo.

En otras ocasiones se requiere dividir al conjunto de datos en **cien porcentajes iguales** y entonces cuando **P “ 0.01,0.02,...,0.99** a los cuantiles correspondientes se les llama **PERCENTILES**.

In [48]:
percentil = np.percentile(edad,90)
percentil

80.0